ormar/queryset/queryset.py

```diff
@@ -1127,5 +1132,11 @@ async def bulk_update(  # noqa:  CCR001
         expr = str(expr)
         await self.database.execute_many(expr, ready_objects)
 
-        for objt in objects:
-            objt.set_save_status(True)
+        for obj in objects:
+            obj.set_save_status(True)
+
+        entity = list(objects)[0]
+        await entity.signals.post_bulk_update.send(
+            sender=entity.__class__, instances=objects
```

Comment:
Use `self.__class__` instead of extracting the entity from a list. 

Since as of now `ormar` does not support onupdate database triggers there is no need for refresh from db, so should be good to go.

Reply:
maybe should support

ormar.datetime(onupdate=datetime.now)

Reply:
That can be considered in other PR

Reply:
yes.  I think it will be supported later.

Reply:
Sorry it should be `self.model_cls`, forgot we are in `QuerySet` not `Model`.